parlai/crowdsourcing/tasks/turn_annotations/bot_agent.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import copy
+import os
+
+from omegaconf import DictConfig
+import parlai.utils.logging as logging
+from parlai.core.agents import create_agent
+from parlai.core.opt import Opt
+from parlai.utils.strings import normalize_reply
+from parlai.crowdsourcing.tasks.turn_annotations.constants import AGENT_1
+from parlai.crowdsourcing.tasks.turn_annotations.utils import Compatibility
+
+
+class TurkLikeAgent:
```

Comment:
Hmm - you don't think `TurkLikeAgent` is needed anymore? Do you know what can be used in its place?

Reply:
Well now the bot agent has its own thing, so the logic for getting an act from it can be in the world. If there's a specified bot and human agent you don't need to treat the former like a turk agent.